iina/Base.lproj/Localizable.strings

```diff
@@ -245,6 +245,8 @@
 "alert.error_saving_file" = "Error occurred when saving %@: %@";
 "alert.error_deleting_file" = "Cannot delete the file.";
 
+"alert.target_peak.bad_value" = "The value \"%@\" is invalid.\nPlease provide a value betwen 10 and 10000, or 0.";
```

Comment:
Between. Not sure why we have a typos CI job if it can’t catch these…

Reply:
I think that only checks for typos in code and comments